install.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Installer for Lingua Franca CLI tools.
+# 
+# Usage: curl -s https://lf-lang.org/install-cli.sh | bash -s nightly
+
+if [[ $1 == "nightly" ]]; then
+  kind="nightly"
+  rel="https://api.github.com/repos/lf-lang/lingua-franca/releases/tags/nightly"
```

Comment:
It seems like a bad idea to download stuff from the internet and then install it unchecked. There should be at least a checksum validation step. This, however, would require to also release the install script nightly with the checksums updated.

Reply:
Isn't this covered by the fact that we use HTTPS already?

Reply:
No, because we don't validate the certificate. Anyone could redirect this request by manipulating the DNS for instance.

Reply:
Then let's check the certificate?

Reply:
I realized that I was wrong about that. Https protects against DNS spoofing. If we trust the github API to deliver unmanipulated artifacts and all LF maintainers, then checksums don't provide additional protection.